lib/Support/MSFileSystemBasic.cpp

```diff
@@ -125,7 +125,7 @@ struct MSFileSystemHandle
   {
   }
 
-  MSFileSystemHandle(IEnumSTATSTG *pEnumSTATG)
+  explicit MSFileSystemHandle(IEnumSTATSTG *pEnumSTATG)
```

Comment:
This isn't mentioned in the commit message. Is it strictly necessary? Additionally, I don't see any instance of a pointer being assigned false, which was mentioned in the commit message

Reply:
Just don't want surprisingly construct a MSFileSystemHandle object from a IEnumSTATSTG pointer. Using 'explicit' on any single parameter constructor is a good convention. If implicit construct is intentional, it needs to be commented.

Reply:
The commit rebased and update the message.

The pointer = false has been fixed some time ago.